frontend/tests/unit/calculators/census-tracts-test.js

```diff
@@ -299,25 +310,90 @@ module('Unit | Calculator | census-tracts', function(hooks) {
       },
     ];
 
-    const newTctCalc = tctCalc.create({
+    this.newTctCalc = tctCalc.create({
       censusTracts: censusTractVariables,
       modesForAnalysis,
     });
+  });
 
+  test('it calculates totalCounts', function(assert) {
     // for what totalCount is, see
     // /frontend/app/calculators/transportation/census-tracts.js
-    assert.equal(newTctCalc.totalCount,
-      autoCensusTract1
-      + autoCensusTract2
-      + taxiCensusTract1
-      + taxiCensusTract2
-      + busCensusTract1
-      + busCensusTract2
-      + subwayCensusTract1
-      + subwayCensusTract2
-      + walkCensusTract1
-      + walkCensusTract2
-      + railCensusTract1
-      + railCensusTract2);
+    assert.equal(this.newTctCalc.totalCount,
+      this.autoCensusTract1
+      + this.autoCensusTract2
+      + this.taxiCensusTract1
+      + this.taxiCensusTract2
+      + this.busCensusTract1
+      + this.busCensusTract2
+      + this.subwayCensusTract1
+      + this.subwayCensusTract2
+      + this.walkCensusTract1
+      + this.walkCensusTract2
+      + this.railCensusTract1
+      + this.railCensusTract2);
+  });
+
+  test('it calculates modeSplits', function(assert) {
+    const { totalCount } = this.newTctCalc;
+
+    const autoCount = this.autoCensusTract1 + this.autoCensusTract2;
+    const taxiCount = this.taxiCensusTract1 + this.taxiCensusTract2;
+    const busCount = this.busCensusTract1 + this.busCensusTract2;
+    const subwayCount = this.subwayCensusTract1 + this.subwayCensusTract2;
+    const railCount = this.railCensusTract1 + this.railCensusTract2;
+    const walkCount = this.walkCensusTract1 + this.walkCensusTract2;
+    const ferryCount = this.ferryCensusTract1 + this.ferryCensusTract2;
+    const streetcarCount = this.streetcarCensusTract1 + this.streetcarCensusTract2;
+    const bicycleCount = this.bicycleCensusTract1 + this.bicycleCensusTract2;
+    const motorcycleCount = this.motorcycleCensusTract1 + this.motorcycleCensusTract2;
+    const otherCount = this.otherCensusTract1 + this.otherCensusTract2;
+
+    assert.deepEqual(this.newTctCalc.modeSplits, {
+      auto: {
+        allPeriods: round(100 * (autoCount / totalCount), 1),
```

Comment:
I would rather use hard-coded values to assert against rather than re-creating the same math in the test that's supposed to test the math

Reply:
Yeah I had a yucky feeling when I wrote this. I think you're right!